app/src/main/res/layout/item_product_category.xml

```diff
@@ -67,7 +68,7 @@
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_marginTop="@dimen/dimen_item_product_category_price_top_margin"
-                android:text="@{product.price + '$'}"
+                android:text="@{product.price + @string/currency}"
```

Comment:
Please use parametrized string 
Here's examples:
https://developer.android.com/guide/topics/resources/string-resource#formatting-strings
https://stackoverflow.com/questions/38978499/how-do-i-use-databinding-to-combine-a-string-from-resources-with-a-dynamic-varia

Reply:
fixed